.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,137 @@
+name: "\U0001F41E Report a problem with MDN Plus"
+description: Report a problem with MDN Plus.
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: dropdown
+    id: country
+    attributes:
+      label: Country
+      description: From what country are you accessing MDN Plus?
+      options:
+        - United States
+        - Canada
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: feature
+    attributes:
+      label: Feature
+      description: What MDN Plus feature is your bug report about?
+      options:
+        - Subscription
+        - Notifications
+        - Collections
+        - MDN Offline
+        - Other
```

Comment:
Could we make this an open text box - it could be they have a feature suggestion instead of something about an existing